Added Steam related "live" scam domains to the list.

PoorPocketsMcNewHold opened the pull request:
So, Here's the additional details. 
I currently maintain a [Steam Scam Sites](https://github.com/PoorPocketsMcNewHold/SteamScamSites) related domains list, mainly aimed at Steam, but also Steam games related phishing attempt (Mostly trading/gambling items websites on several games like Counter-Strike: Global Offensive, Team Fortress 2, Dota 2, PlayerUnknown´s Battlegrounds)
But anyway, You already got all that in #63.
Here're the domains from my list which aren't in your list, and domains still alive.
As i'm mainly still beginner in such job, I don’t really have an extreme method for checking domains availability sadly. I’ve had before the link to a bulk domain lookup service, in which let me decide on each these websites according to their response from the sent requests, but i since now just verify if those domains are still used (Meaning, if they still got an IP address to access to them, despite if the page is active, broken, sold, or just plan blank). So, feel free to curate those domains below from my pull request.

PoorPocketsMcNewHold committed:
Added Steam related "live" scam domains to the list.

More details will be added later, when i'm home.

SvicidalBug commented:
Nice collection! Some things I noticed should be changed first.

- Typically, domains that appear to be dead should [still be considered threats until a new owner buys them and uses it for a legit purpose.](https://github.com/HexxiumCreations/threat-list/pull/36#issuecomment-402500611)

- 'www.' should be removed from the beginning of each domain.

- cs.money is a legitimate trading site for CS:GO

- Although tf2hunt.com's user base is largely made up of scammers, it is a legitimate TF2 gambling site.

- skinhub.com looks like a legitimate... crate opening site? I know next to nothing about CS:GO trading, but I thought crates were to be opened in-game. Anyway, [they've got a Twitter page](https://twitter.com/skinhub) with quite a bit of proof that suggests they are not a scam site.

- steampowers.net/gametame.com is a legitimate site that provides offer walls where you can earn points and redeem them for items on Steam. 

- g2a.com itself is not a scam site, just that some sellers on there sell stolen Steam keys.

- pinnacle.com/pinnaclesports.com is a legitimate sports betting website.

- steamanalyst.com is a legitimate site that tracks CS:GO item prices on several different trading websites.

- handouts.gg looks like a GPT site similar to gametame.com

- clc.am is a link shortener

- takethestairs.today doesn't seem like a threat. Is there a phishing page on that site somewhere maybe?

I haven't checked all of them, these are just the ones that stood out to me.

If you happen to be unsure if a site is a scam, a quick web search of "*site name* scam" might find a Steam discussion thread or SteamRep report talking about it. For phishing, its easy to tell since Steam will never ask for your log in credentials if you are already logged in.

Besides that, this is good start. Thanks for your contribution!

PoorPocketsMcNewHold committed:
Added found dead domains

PoorPocketsMcNewHold commented:
I was aware of, possible false flag domains, but didn’t manage to found these.

> Typically, domains that appear to be dead should still be considered threats until a new owner buys them and uses it for a legit purpose.

Added the rest of my dead domains from my list to it.

> 'www.' should be removed from the beginning of each domain.

I usually typically do that, that was something i’ve forgotten to remove for a few.

> cs.money is a legitimate trading site for CS:GO

> skinhub.com looks like a legitimate... crate opening site? I know next to nothing about CS:GO trading, but I thought crates were to be opened in-game. Anyway, they've got a Twitter page with quite a bit of proof that suggests they are not a scam site.

Original source mistake. Gonna remove it temporary and see why its addition to the main repo. I suppose, coupon code spam issues, like most of the false flags there.

> Although tf2hunt.com's user base is largely made up of scammers, it is a legitimate TF2 gambling site.

> pinnacle.com/pinnaclesports.com is a legitimate sports betting website.

> takethestairs.today doesn't seem like a threat. Is there a phishing page on that site somewhere maybe?

https://github.com/PoorPocketsMcNewHold/SteamScamSites/commit/4a740dba2b62ade795831001d422539b7467d617 Redirected from other phishing domains, why would there finally redirect it too there? No idea, but i’ve added it anyway. Will remove it, as the Steam login page seems valid.

> steampowers.net/gametame.com is a legitimate site that provides offer walls where you can earn points and redeem them for items on Steam.

> g2a.com itself is not a scam site, just that some sellers on there sell stolen Steam keys.

Those two come from the original list once again, but I might see the issue with this, steampowers.net/gametame.com obtaining their keys from gray market, with no warranty of the claimed games (or REWARDS as described) to be working, and well, g2a in itself is a pure gray market.
Even if i’m really willing to not recommend those, it's true that it shouldn't be in the list in the first place. I’ll apply the remove temporary/ask main source rule.
```
Disclaimer of Warranties
Member expressly agrees that use of GameTame is at Member's sole risk.

GameTame and all services offered therein are provided on a strictly "as is" and "as available" basis. gametame.com MAKES NO WARRANTY WITH REGARD TO ANY PRODUCTS, SERVICES, OR REWARDS OBTAINED BY MEMBERS THROUGH GameTame OR THROUGH GameTame ADVERTISERS OR PARTNERS; THAT GameTame WILL MEET MEMBERS' REQUIREMENTS; OR THAT GameTame WILL BE UNINTERRUPTED, TIMELY, ERROR FREE; NOR DOES gametame.com MAKE ANY WARRANTY AS TO THE RESULTS THAT MAY BE OBTAINED FROM THE USE OF GameTame OR AS TO THE ACCURACY OR RELIABILITY OF ANY INFORMATION OBTAINED THROUGH GameTame.

gametame.com expressly disclaims all express and implied warranties, including, but not limited to, the implied warranties of merchantability, fitness for a particular purpose, and non-infringement. gametame.com shall not be liable or responsible for those guarantees, warranties, and representations, if any, offered by GameTame advertisers, partners, manufacturers of merchandise, or suppliers of services. No advice or information, whether oral or written, obtained by Member from gametame.com or through GameTame shall create any warranty not expressly made herein.
```

> clc.am is a link shortener

> steamanalyst.com is a legitimate site that tracks CS:GO item prices on several different trading websites.

Mistake of mine with one of the sources. Will remove it.

I'll put a [W.I.P] tag on that pull request, to allow me to do cleaner up and a couple of research on the current domains that i'm having.

SvicidalBug commented:
Thanks. More on GameTame, I had used them awhile back and can confirm [they always did pay me.](https://i.imgur.com/NRVOkMV.png) Their offer walls though are [riddled](https://i.imgur.com/HjlplKW.png) with [scams](https://i.imgur.com/ReBHMxA.png) that never actually credit points to your account. I don't use them anymore, and I wouldn't recommenced anyone else use them either. I just don't think it would be fair to block them because advertisers are being shitty.

> Redirected from other phishing domains, why would there finally redirect it too there? No idea, but i’ve added it anyway.

It's possible those sites were offering some sort of referral benefit, like a store credit or free item. But that wouldn't explain takethestairs.today ¯\_(ツ)_/¯

PoorPocketsMcNewHold committed:
Removed the only domains needed to be removed.

PoorPocketsMcNewHold committed:
And, corrected a little mistake.

PoorPocketsMcNewHold commented:
So, I’m speculating on that one, but this would be pretty legit explanation for takethestairs.today.
Well, here's what some do with their domains when it isn’t set on phishing mode.
They either blank page it, make it offline of course, or reuse it for a completely out of context purpose, mainly just re hosting some other website source.
Here's for example, one of the domains i’ve caught in my list, blatantly just copy pasting a website for... no reasons it seems?
So, I’ll prefer to keep those random domains in the list in case that they are reused.
![image](https://user-images.githubusercontent.com/13170204/57584463-d0b9ad00-74db-11e9-95cc-bc30aff5f9a1.png)

Outside of that, I didn’t found any real issues in other domains i’ve checked.
Some points I would like to point at :
- Despite some sites having legit steam logins, It can still be a scam website. By making you betting on your items, for a tricked impossible bet. This can habitually been proven by checking the actual players if they do exist, and if they do actually use this website. Those websites can also been used with scam attempts directly to steam accounts, by the ¨I´m an admin for X¨ scam, saying that it will raise the chance for you only.
- For some strange reasons, a lot of website didn’t declare they’re registration identities, making it pretty difficult if a certain website is just a basic legit one, or a scam website. But I expect actual real betting website to have a bit more details added to their server certificate for example.

PoorPocketsMcNewHold committed:
Added back the stairs once again.

Hexxium commented:
Will take a look over this soon and merge it, I just haven't had the time yet, might still be a short bit until my next day off

PoorPocketsMcNewHold commented:
That's fine. I took a long time before updating it anyway. If needed, I've had save a excel file of the domains whois results. If needed, I can share it later on.

-------- Message d'origine --------
On 12 mai 2019 à 21:04, Hexxium a écrit :

> Will take a look over this soon and merge it, I just haven't had the time yet, might still be a short bit until my next day off
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, [view it on GitHub](https://github.com/HexxiumCreations/threat-list/pull/64#issuecomment-491620567), or [mute the thread](https://github.com/notifications/unsubscribe-auth/ADEPMHGP4EE67JNC36AYWRDPVBS4TANCNFSM4HLG6A3A).

Hexxium committed:
Readds removed domains

per https://github.com/HexxiumCreations/threat-list/pull/64

Hexxium commented:
Re-added the following:

giftcardrebel.com
getsteambonus.com
getsteamgifts.com
efsjdomongowalo.xyz
guccidrop.top
officialkmspico.com
steamcodegenerator.xyz
windatgamex.com	
spindatgamex.com
mygiveawayitem.com

They appear to be removed in your commit history and some are still active threats, blocked by an antivirus, and in general threats in the list should [still be considered threats until a new owner buys them and uses it for a legit purpose.](https://github.com/HexxiumCreations/threat-list/pull/36#issuecomment-402500611) The rest of the commits looked okay, I've gone through most of them.

I'll revise the commit guidelines soon since things have changed since I last wrote them, ideally going forward please commit to threat-list using its format rather than the domainsonly file, its much faster for me to add the result there to the others by auto removing the symbols, I'll merge this commit if there is no further changes pending
